lib/nomad-util/src/lib.rs

```diff
@@ -10,9 +10,20 @@ use nomad_client::apis::configuration::Configuration;
 pub use crate::error::NomadError;
 
 pub fn config_from_env() -> Result<Configuration, NomadError> {
-	let nomad_url =
-		std::env::var("NOMAD_URL").map_err(|_| NomadError::MissingEnvVar("NOMAD_URL".into()))?;
-	let config = nomad_client::apis::configuration::Configuration {
+	let nomad_url = std::env::var("NOMAD_URL")
+		.map_err(|_| NomadError::MissingEnvVar("NOMAD_URL".into()))?;
+	let config = Configuration {
+		base_path: format!("{}/v1", nomad_url),
+		..Default::default()
+	};
+
+	Ok(config)
+}
+
+pub fn new_config_from_env() -> Result<nomad_client_new::apis::configuration::Configuration, NomadError> {
```

Comment:
nit: "new config" will be what all new code is written with. can we rename the old config_from_env to deprecated_config_from_env or something like that?